client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -62,6 +66,14 @@ const MediaModalSecondaryActions = React.createClass( {
 		this.props.onChangeView( ModalViews.DETAIL );
 	},
 
+	navigateToPlans() {
+		analytics.ga.recordEvent( 'Media', 'Clicked Plan Storage Button' );
+		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', {
+			cta_name: 'plan-storage'
```

Comment:
We might want to further namespace this, like `plan-media-storage`. I think if/when we add a CTA in the warning we'd probably want to track that separately as well.